CI failing

A few are from MDTraj no longer being packaged by the toolkit by default, easy fix

Most frequent comes from setup

```
../../../micromamba/envs/qcsubmit-test-psi4/lib/python3.12/site-packages/_pytest/config/__init__.py:1892: AttributeError

The above exception was the direct cause of the following exception:

pytestconfig = <_pytest.config.Config object at 0x7f188b797d70>

    @pytest.fixture(scope="function")
    def fulltest_client(pytestconfig):
        """
        A portal client used for full end-to-end tests
        """
    
>       uri = pytestconfig.getoption("--fractal-uri")
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
E       ValueError: no option named '--fractal-uri'
```

https://github.com/openforcefield/openff-qcsubmit/actions/runs/27245917977

--- a/.readthedocs.yaml
+++ b/.readthedocs.yaml
@@ -1,7 +1,7 @@
 version: 2
 
 build:
-  os: ubuntu-26.04
+  os: ubuntu-24.04
   tools:
     python: "mambaforge-latest"
   jobs:
@@ -20,5 +20,3 @@ sphinx:
   configuration: docs/conf.py
   fail_on_warning: false
 
-formats:
-- pdf

--- a/.readthedocs.yaml
+++ b/.readthedocs.yaml
@@ -3,7 +3,10 @@ version: 2
 build:
   os: ubuntu-26.04
   tools:
-    python: "mambaforge-4.10"
+    python: "mambaforge-latest"
+  jobs:
+    pre_build:
+      - mamba list
 
 python:
   install:

--- a/.readthedocs.yaml
+++ b/.readthedocs.yaml
@@ -1,7 +1,7 @@
 version: 2
 
 build:
-  os: ubuntu-20.04
+  os: ubuntu-26.04
   tools:
     python: "mambaforge-4.10"
 

--- a/.readthedocs.yaml
+++ b/.readthedocs.yaml
@@ -4,10 +4,6 @@ build:
   os: ubuntu-24.04
   tools:
     python: "mambaforge-latest"
-  jobs:
-    pre_build:
-      - mamba list
-
 python:
   install:
     - method: pip

--- a/devtools/conda-envs/basic.yaml
+++ b/devtools/conda-envs/basic.yaml
@@ -25,7 +25,7 @@ dependencies:
 
   - qcengine >=0.25
   - qcelemental >=0.25.1
-  - qcfractal >=0.65
+  - qcfractal ==0.56
   - qcarchivetesting
   - qcportal
 

--- a/devtools/conda-envs/basic.yaml
+++ b/devtools/conda-envs/basic.yaml
@@ -25,7 +25,7 @@ dependencies:
 
   - qcengine >=0.25
   - qcelemental >=0.25.1
-  - qcfractal >=0.56
+  - qcfractal >=0.65
   - qcarchivetesting
   - qcportal
 
@@ -52,3 +52,4 @@ dependencies:
   # Optional
   - openmmforcefields ~=0.14.2
   - openff-fragmenter-base >=0.2.0
+  - mdtraj

--- a/devtools/conda-envs/basic.yaml
+++ b/devtools/conda-envs/basic.yaml
@@ -25,7 +25,7 @@ dependencies:
 
   - qcengine >=0.25
   - qcelemental >=0.25.1
-  - qcfractal ==0.56
+  - qcfractal >=0.65
   - qcarchivetesting
   - qcportal
 

--- a/devtools/conda-envs/psi4.yaml
+++ b/devtools/conda-envs/psi4.yaml
@@ -24,7 +24,7 @@ dependencies:
 
   - qcengine >=0.25
   - qcelemental >=0.25.1
-  - qcfractal >=0.56
+  - qcfractal >=0.65
   - qcarchivetesting
   - qcportal
 
@@ -50,5 +50,6 @@ dependencies:
   - openmmforcefields ~=0.14.2
   - openff-fragmenter-base >=0.2.0
   - openmm
+  - mdtraj
 
   - postgresql

--- a/devtools/conda-envs/psi4.yaml
+++ b/devtools/conda-envs/psi4.yaml
@@ -24,7 +24,7 @@ dependencies:
 
   - qcengine >=0.25
   - qcelemental >=0.25.1
-  - qcfractal >=0.65
+  - qcfractal ==0.56
   - qcarchivetesting
   - qcportal
 

--- a/devtools/conda-envs/psi4.yaml
+++ b/devtools/conda-envs/psi4.yaml
@@ -24,7 +24,7 @@ dependencies:
 
   - qcengine >=0.25
   - qcelemental >=0.25.1
-  - qcfractal ==0.56
+  - qcfractal >=0.65
   - qcarchivetesting
   - qcportal
 

--- a/docs/releasehistory.md
+++ b/docs/releasehistory.md
@@ -207,7 +207,7 @@ For more information on this release, see https://github.com/openforcefield/open
 [PR #318:]: https://github.com/openforcefield/openff-qcsubmit/pull/318
 [PR #340:]: https://github.com/openforcefield/openff-qcsubmit/pull/340
 [PR #367:]: https://github.com/openforcefield/openff-qcsubmit/pull/367
-[PR #283:]: https://github.com/openforcefield/openff-qcsubmit/pull/382
+[PR #382:]: https://github.com/openforcefield/openff-qcsubmit/pull/382
 
 [@jthorton]: https://github.com/jthorton
 [@dotsdl]: https://github.com/dotsdl

--- a/docs/releasehistory.md
+++ b/docs/releasehistory.md
@@ -14,6 +14,8 @@ Releases are given with dates in DD-MM-YYYY format.
 ## Current development
 
 * [PR #367:] Do not use `pkg_resources`
+* [PR #382:] Update tests for deprecation of fulltest_client in QC* 0.56 and removal of mdtraj as a hard dependency of openff-toolkit.
+
 
 ## 0.57.0 / 10-02-2025
 
@@ -205,6 +207,7 @@ For more information on this release, see https://github.com/openforcefield/open
 [PR #318:]: https://github.com/openforcefield/openff-qcsubmit/pull/318
 [PR #340:]: https://github.com/openforcefield/openff-qcsubmit/pull/340
 [PR #367:]: https://github.com/openforcefield/openff-qcsubmit/pull/367
+[PR #283:]: https://github.com/openforcefield/openff-qcsubmit/pull/382
 
 [@jthorton]: https://github.com/jthorton
 [@dotsdl]: https://github.com/dotsdl

--- a/openff/qcsubmit/_tests/conftest.py
+++ b/openff/qcsubmit/_tests/conftest.py
@@ -13,6 +13,25 @@ def public_client():
     return PortalClient("https://api.qcarchive.molssi.org:443/")
 
 
+@pytest.fixture(scope="function")
+def fulltest_client():
+    """A portal client backed by a local FractalSnowflake for full end-to-end tests.
+
+    This shadows the ``fulltest_client`` fixture provided by the ``qcarchivetesting``
+    pytest plugin. As of qcarchivetesting 0.65 that fixture reads the ``--fractal-uri``
+    option, but the plugin that registered the option was removed, so the upstream
+    fixture raises ``ValueError: no option named '--fractal-uri'``. We only ever use
+    the default (a local snowflake), so build it directly here.
+    """
+    from qcfractal.snowflake import FractalSnowflake
+
+    snowflake = FractalSnowflake()
+    try:
+        yield snowflake.client()
+    finally:
+        snowflake.stop()
+
+
 @pytest.fixture()
 def imatinib_mesylate() -> Molecule:
     return Molecule.from_file(get_data("imatinib_mesylate.sdf"))

